pylib/anki/collection.py

```diff
@@ -420,6 +420,13 @@ def import_json_file(self, path: str) -> ImportLogWithChanges:
     def import_json_string(self, json: str) -> ImportLogWithChanges:
         return self._backend.import_json_string(json)
 
+    def export_dataset_for_research(
+        self, min_entries: int = 0, target_path: str | None = None
+    ) -> None:
+        if target_path is None:
+            target_path = os.path.join(os.path.dirname(self.path), "dataset.pb")
```

Comment:
I'd prefer target_path was required instead of falling back to dataset.pb. Otherwise looks good :+1:

Reply:
Thanks for the advice. I update it.